my-app/src/main/java/com/JavaWebServer/app/Request.java

```diff
@@ -10,11 +10,30 @@ public Request () {
 
   }
 
+  public boolean isParams() {
+    return this.request.split(" ")[1].contains("?");
+  }
+
   public String getRoute() {
-    return this.request.split(" ")[1];
+    if (isParams()){
+      return getParamsRoute();
+    } else {
+      return this.request.split(" ")[1];
+    }
   }
 
+  public String getParams() {
+    return request.split(" ")[1].split("\\?")[1];
```

Comment:
What does this space + `\\?` refer to? Maybe you should have some variables to help make this more readable.